worldedit_commands/locale/template.txt

```diff
@@ -148,4 +148,8 @@ Region expanded by @1 nodes=
 Contracts the selection in the selected absolute or relative axis=
 Region contracted by @1 nodes=
 Select a cube with side length <size> around position 1 and run <command> on region=
+invalid usage: //@1 cannot be used with cubeapply=
 Missing privileges: @1=
+
+### wand.lua ###
+WorldEdit Wand tool@\nLeft-click to set 1st position, right-click to set 2nd=
```

Comment:
has to be `@n`

Reply:
https://minetest.gitlab.io/minetest/translations/#escapes
> - `@\n` (where the \n is a literal newline) acts as a literal newline. As with `@=`, this escape is not required in strings given to `minetest.translate`, but is in translation files.
> - `@n` acts as a literal newline as well.

You can use both. But!!, with `@n` not works as expected (maybe I made a mistake somewhere).
Anyway by documentation you can use both. And this works as expected.
Let's leave it as is.